stacks-signer/src/cli.rs

```diff
@@ -260,12 +264,26 @@ fn parse_contract(contract: &str) -> Result<QualifiedContractIdentifier, String>
     QualifiedContractIdentifier::parse(contract).map_err(|e| format!("Invalid contract: {}", e))
 }
 
-/// Parse a BTC address argument and return a `PoxAddress`
+/// Parse a BTC address argument and return a `PoxAddress`.
+/// This function behaves similarly to `PoxAddress::from_b58`, but also handles
+/// addresses where the parsed AddressHashMode is None.
 pub fn parse_pox_addr(pox_address_literal: &str) -> Result<PoxAddress, String> {
-    PoxAddress::from_b58(pox_address_literal).map_or_else(
+    let parsed_addr = PoxAddress::from_b58(pox_address_literal).map_or_else(
         || Err(format!("Invalid pox address: {pox_address_literal}")),
         |pox_address| Ok(pox_address),
```

Comment:
This is a total nit and I think I fixed this in another pr, but you can change `|pox_address| Ok(pox_address)` to just `Ok`

Reply:
Ah, love that! Fixed